python/veles/db/backend.py

```diff
@@ -523,7 +523,8 @@ def list(self, parent, tags=frozenset(), pos_filter=PosFilter()):
         return {NodeID(bytes(x)) for x, in c.fetchall()}
 
     def begin(self):
-        assert not self.db.in_transaction
+        if six.PY3:
```

Comment:
Hmm, now that I think about it - this is server code, we decided that server will only run on Py3 - then why do we need this check?